src/Employer/Employer.Web/ViewModels/CreateVacancyOptions/CreateVacancyOptionsEditModel.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Esfa.Recruit.Employer.Web.ViewModels.CreateVacancyOptions
+{
+    public class CreateVacancyOptionsEditModel : CreateVacancyOptionsViewModel
```

Comment:
Is this inheritance the right way round?

Reply:
Nope!!

Reply:
there shouldn't be any inheritance and are we not posting all input values as strings and then using type coersion??